### ISSUE TITLE
how to convert .pth model into .h5 format? or how can we use .pth weight as api

### ISSUE THREAD

**Issue by akashAD98** [Deci-AI/super-gradients#533]
how can we use .pth or .onnx weight ,there inference & want to use this weight in flask,gradio, etc apps

**Comment by lkdci** [comment]
Hi @akashAD98 , converting pytorch models to keras isn't such a trivial thing to do.
I can recommend a really nice repo https://github.com/MPolaris/onnx2tflite which convert an ONNX model to keras h5 model and tflite model. This repo handle this the best among other repos as far as I know.
According to their doc you can see that most of the common classification models are supported and tested, but this tool is still experimental and some segmentation / detection models might not be convert well and might result with discrepancy in output tensors.
